src/main/jte/bookings.jte

```diff
@@ -36,8 +37,8 @@
                     <tr>
                         <td>${booking.getId()} ${booking.getOutstandingRequests().isEmpty() ? context.empty() : context.localize("bookings.outstanding", booking.getOutstandingRequests().size())}</td>
                         <td>${booking.getDescription()}</td>
-                        <td>${booking.getStartDate().toString()}</td>
-                        <td>${booking.getEndDate().toString()}</td>
+                        <td>${booking.getStartDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"))}</td>
+                        <td>${booking.getEndDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"))}</td>
```

Comment:
Do we want this pattern?
Perhaps a localized pattern would be nicer (IIRC Spring provides something for this)

Reply:
What do you mean by localized?
Do you want me to use @DateTimeFormat(pattern = "yyyy-MM-dd") so that DateTimeformatter uses the same format in bookings.jte?

Reply:
We have the current locale (it is wrapped in the context).
It is possible to use this to convert a LocalDateTime to a string in the correct format for the users' locale. The necessary logic for this should be moved to the Context and used consistently within templates.